modules/dedicated-storage-expanding-filesystem-pvc.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * storage/expanding-persistent-volume.adoc
+
+[id="expanding-pvc-filesystem_{context}"]
+= Expanding {product-name} Persistent Volume Claims (PVCs)
+
+Expanding PVCs based on volume types that need file system resizing,
+such as AWS EBS, is a two-step process.
+This process involves expanding volume objects in the cloud provider, and
+then expanding the file system on the actual node. These steps occur automatically
+after the PVC object is edited, and may require a pod restart to take effect.
+
+Expanding the file system on the node only happens when a new pod is started
+with the volume.
+
+.Prerequisites
+
+* The controlling StorageClass must have `allowVolumeExpansion` set
+to `true`. This is the default configuration in {product-name}.
+
++
+[IMPORTANT]
+====
+Decreasing the size of an AWS EBS volume is not supported. However, you
+can create a smaller volume and then migrate your data to it using a
+tool such as `oc rsync`. After modifying a volume, you must wait at least six hours before
+making additional modifications to the same volume.
+====
+
+.Procedure
+
+. Edit the PVC and request a new size by editing
+`spec.resources.requests.storage`. For example, the following expands
+the `ebs` PVC to 8 Gi.
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: ebs
+spec:
+  storageClass: "gp2"
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi <1>
+----
+<1> Updating `spec.resources.requests.storage` to a larger amount will expand
+the PVC.
++
+You can do this with a simple `oc patch` command:
++
+----
+$ oc patch pvc ebs -p '{"spec": {"resources": {"requests": {"storage": "8Gi"}}}}'
+----
+
+. Once the cloud provider object has finished resizing, the PVC is set to
+`FileSystemResizePending`. The following command is used to check
+the condition:
++
+----
+$ oc describe pvc <pvc_name>
+----
++
+This condition may resolve automatically, or in some cases you will see
+a message indicating that the pod will need to be restarted first:
++
+----
+Conditions:
+  Type                     Status  LastProbeTime      LastTransitionTime  Reason  Message
+  ----                     ------  -----------------  ------------------  ------  -------
+  FileSystemResizePending  True    <Timestamp>        <Timestamp>                 Waiting for user to (re-)start a pod to
+                                                                                  finish file system resize of volume on node.
+----
++
+When you see this specific message, you will need to restart the mounting pod. This is
+also exposed in the `describe` command above:
++
+----
+Mounted By:  mysql-1-2cb7q
+----
++
+The quickest way to do this is by deleting the pod and letting {product-name}
+provision a replacement pod.
+
+. Once the pod is running, the newly requested size is available and the
```

Comment:
Please either rework this step to be a note that's attached to the last step or repeat the describe command and tell the user to confirm that the condition is gone.